hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/ScmConfigKeys.java

```diff
@@ -490,6 +490,14 @@
   public static final boolean HDDS_DATANODE_UPGRADE_LAYOUT_INLINE_DEFAULT =
       true;
 
+
+  // Temporary config which will be used only for test only purposes until
+  // SCM HA Security work is completed. This config should not be modified by
+  // users.
+  public static final String OZONE_SCM_HA_SECURITY_ENABLE =
```

Comment:
NIT: can we rename this to OZONE_SCM_HA_SECURITY_SUPPORTED as this is compared against other keys has defined for security enabled and HA enabled? Otherwise, LGTM.

Reply:
Done